src/lib/DepositApiClient.js

```diff
@@ -118,6 +118,8 @@ export class DepositApiClient {
     );
   }
 
+  // TODO: Might consider extracting these out to a FilesApiClient.js
```

Comment:
comment: I didn't do that as it was just one function and also the depositApiClient is configurable so in case you want to override the class you just have everything there...maybe instead of TODO create an issue? Easier to track them :)

Reply:
:+1: I've added an issue here: https://github.com/inveniosoftware/react-invenio-deposit/issues/157 . This was a case of a little piece of thread that unraveled much more once I pulled on it.